code/modules/admin/verbs/adminjump.dm

```diff
@@ -1,23 +1,56 @@
-/client/proc/Jump(area/A in return_sorted_areas())
-	set name = "Jump to Area"
-	set desc = "Area to jump to"
+/client/verb/jump_to()
+	set name = "Jump to..."
```

Comment:
Making this `/client/verb` makes it accessible to everyone regardless of rank. They wont be able to use it but it will be misleading.

Reply:
![image](https://user-images.githubusercontent.com/25063394/108601612-49c68380-7395-11eb-8e6d-e1f42b51cf5f.png)

Reply:
I'm not able to reproduce it showing up there, but clearly it does so...